ctx_test.go

```diff
@@ -27,8 +28,10 @@ import (
 	"text/template"
 	"time"
 
+	"github.com/andybalholm/brotli"
 	"github.com/gofiber/fiber/v3/internal/storage/memory"
 	"github.com/gofiber/utils/v2"
+	"github.com/klauspost/compress/zstd"
```

Comment:
Why did this get added?

Reply:
If you run go mod tidy, these will get added to go.mod outside of fasthttp which is not ideal. See below.